core/src/main/scala/Monitoring.scala

```diff
@@ -176,8 +202,10 @@ trait Monitoring {
           Task.delay(logErrors(Task.fork(receivedIdempotent.run)(defaultPool)).runAsync(_ => ()))
         }
         case Discard(source) => for {
-          _ <- Task.delay { log.info(s"Attempting to stop monitoring $source...") }
-          _ <- Option(urlSignals.get(source)).traverse_(_.close)
+          _ <- Task.delay {log.info(s"Attempting to stop monitoring $source...") }
+          _ <- Option(urlSignals.get(source)).
```

Comment:
To clarify, its weird with the trailing `.` - we dont do that in any of our codebases and its not typical scala programming style.